onnxruntime/core/framework/error_code_helper.h

```diff
@@ -8,3 +8,10 @@
 namespace onnxruntime {
 OrtStatus* ToOrtStatus(const onnxruntime::common::Status& st);
 };
+
+#define API_IMPL_BEGIN try {
+#define API_IMPL_END                                          \
+  }                                                           \
+  catch (std::exception & ex) {                               \
```

Comment:
const std::exception&

Reply:
Good catch, I always catch by const reference but this code was just copied and pasted and I didn't notice it there.